streamer/pulp/streamer/server.py

```diff
@@ -232,6 +229,7 @@ def _download(self, request, entry, responder):
         finally:
             try:
                 downloader.config.finalize()
+                self.session_cache.put()
```

Comment:
Oh, good catch.  Leftover from when items needed to be released back to the cache.